src/util/queue/api.ts

```diff
@@ -194,9 +198,9 @@ async function editTicket(id: string, ownerID: string, queueID: string, status:
 /**
  * Deletes a ticket with the given ID.
  */
-async function deleteTicket(id: string, queueID: string): Promise<void> {
+async function deleteTicket(id: string, queueID: string, status: TicketStatus, ta: boolean): Promise<void> {
```

Comment:
nit: please use the `isBoolVal` naming convention for booleans. In this case, `ta` becomes `isTa`